demo/finding.js

```diff
@@ -50,7 +50,7 @@ ebay.findCompletedItems({
 // // This call searches for items on eBay using specific eBay product values.
 // https://developer.ebay.com/DevZone/finding/CallRef/findItemsByProduct.html#findItemsByProduct
 ebay.findItemsByProduct({
-    productId: 53039031,
+    productID: 53039031,
     entriesPerPage: 2
```

Comment:
@TotallyNotChase I don't think you have to change variable names as part of this PR, since this library is out and many people are using it. It would be difficult until or unless we have mention these changes as breaking changes and release a major version.